code/python/src/datastar_py/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .sse import SSE_HEADERS, ServerSentEventGenerator
+
+__ALL__ = [ServerSentEventGenerator, SSE_HEADERS]
```

Comment:
Should probably be

```python
__all__ = ["ServerSentEventGenerator", "SSE_HEADERS"]
```

Before I saw this PR, I created https://github.com/lllama/datastar/pull/1